src/runtime_src/xdp/profile/plugin/aie_profile/edge/aie_profile.cpp

```diff
@@ -123,19 +125,31 @@ namespace xdp {
                                                XAIE_EVENT_DMA_MM2S_1_STALLED_LOCK_ACQUIRE_MEM};
     }
     else {
-      mMemoryStartEvents["dma_stalls_s2mm"] = {XAIE_EVENT_DMA_S2MM_0_STALLED_LOCK_MEM,
-                                               XAIE_EVENT_DMA_S2MM_1_STALLED_LOCK_MEM};
-      mMemoryStartEvents["dma_stalls_mm2s"] = {XAIE_EVENT_DMA_MM2S_0_STALLED_LOCK_MEM,
-                                               XAIE_EVENT_DMA_MM2S_1_STALLED_LOCK_MEM};
+      mMemoryStartEvents["dma_stalls_s2mm"]  = {XAIE_EVENT_DMA_S2MM_0_STALLED_LOCK_MEM,
+                                                XAIE_EVENT_DMA_S2MM_1_STALLED_LOCK_MEM};
+      mMemoryStartEvents["dma_stalls_mm2s"]  = {XAIE_EVENT_DMA_MM2S_0_STALLED_LOCK_MEM,
+                                                XAIE_EVENT_DMA_MM2S_1_STALLED_LOCK_MEM};
+      mMemoryStartEvents["s2mm_throughputs"] = {XAIE_EVENT_DMA_S2MM_0_STALLED_LOCK_MEM,
+                                                XAIE_EVENT_DMA_S2MM_0_MEMORY_BACKPRESSURE_MEM};
+      mMemoryStartEvents["mm2s_throughputs"] = {XAIE_EVENT_DMA_MM2S_0_STREAM_BACKPRESSURE_MEM,
+                                                XAIE_EVENT_DMA_MM2S_0_MEMORY_STARVATION_MEM};
     }
     mMemoryEndEvents = mMemoryStartEvents;
 
     // **** Interface Tile Counters ****
     mShimStartEvents = {
-      {"input_throughputs",       {XAIE_EVENT_PORT_RUNNING_0_PL, XAIE_EVENT_PORT_STALLED_0_PL}},
-      {"output_throughputs",      {XAIE_EVENT_PORT_RUNNING_0_PL, XAIE_EVENT_PORT_STALLED_0_PL}},
-      {"packets",                 {XAIE_EVENT_PORT_TLAST_0_PL,   XAIE_EVENT_PORT_TLAST_1_PL}}
+      {"packets",                 {XAIE_EVENT_PORT_TLAST_0_PL,       XAIE_EVENT_PORT_TLAST_1_PL}},
+      {"input_throughputs",       {XAIE_EVENT_GROUP_DMA_ACTIVITY_PL, XAIE_EVENT_PORT_RUNNING_0_PL}},
+      {"output_throughputs",      {XAIE_EVENT_GROUP_DMA_ACTIVITY_PL, XAIE_EVENT_PORT_RUNNING_0_PL}},
+      {"input_stalls",            {XAIE_EVENT_DMA_MM2S_0_STREAM_BACKPRESSURE_PL, 
```

Comment:
Are these new metric sets added for win implementation as well?

Reply:
Discussed